internal/routes/enx_redirect.go

```diff
@@ -155,36 +158,54 @@ func ENXRedirect(
 		// Using a different name, makes it so cookies don't interfer in local dev.
 		requireSession := middleware.RequireNamedSession(sessions, "en-user-report", h)
 
-		{ // handler for /report/issue, required values must be in the established session.
+		// Setup chaffers
+		issueReportTracker := chaff.New()
+		showReportTracker := chaff.New()
```

Comment:
do we expect clients to chaff both requests?

and unless they're pulling all the assets, this doesn't help. it would still be possible to observe real vs chaff

Reply:
I'm leaning towards not supporting this, let's discuss offline.